druid-shell/src/platform/gtk/window.rs

```diff
@@ -608,6 +609,26 @@ impl WindowBuilder {
                 Inhibit(true)
             }));
 
+        win_state
+            .drawing_area
+            .connect_focus_in_event(clone!(handle => move |_widget, event| {
+                assert!(event.get_in());
```

Comment:
This scares me, partly because the [gtk docs](https://gtk-rs.org/docs/gdk/struct.EventFocus.html#method.get_in) aren't telling me what it does, but mostly because I don't think we should panic just because we got something unexpected from the system. Could it be a `log::error!` instead?

Reply:
I'm happy to remove these.